### PR TITLE
DOC: Simplify install.rst

### PR DESCRIPTION
**Pull request by mroeschke**
* Simplified some preamble
* Updates some links and references
* Removed installing via Linux package managers since this is not managed by the team

### CODE REVIEW

File: doc/source/getting_started/install.rst [hunk 1]
```diff
@@ -30,197 +31,133 @@ Installing pandas
...
 ----------------------
 
-pandas is equipped with an exhaustive set of unit tests, covering about 97% of
-the code base as of this writing. To run it on your machine to verify that
-everything is working (and that you have all of the dependencies, soft and hard,
-installed), make sure you have `pytest
+pandas is equipped with an exhaustive set of unit tests. The unit tests require `pytest
 <https://docs.pytest.org/en/latest/>`__ >= 7.3.2 and `Hypothesis
-<https://hypothesis.readthedocs.io/en/latest/>`__ >= 6.34.2, then run:
+<https://hypothesis.readthedocs.io/en/latest/>`__ >= 6.34.2 to run. To run the tests from a
```
Comment (lithomas1) [inline doc/source/getting_started/install.rst:155]: IIRC you need more than this.

I think you need pytest-asyncio and pytest-localserver(??? not sure if this was removed).

**Review by lithomas1 — COMMENTED** [review]
Nice, this looks like a pretty good simplification.

Left some minor comments.

File: doc/source/getting_started/install.rst [hunk 2]
```diff
@@ -30,197 +31,133 @@ Installing pandas
...
+.. _install.pypi:
 
 Installing from PyPI
 ~~~~~~~~~~~~~~~~~~~~
 
 pandas can be installed via pip from
 `PyPI <https://pypi.org/project/pandas>`__.
 
-.. note::
-    You must have ``pip>=19.3`` to install from PyPI.
```
[the remark this answers is not in the document]
Comment (mroeschke) [reply doc/source/getting_started/install.rst:136]: Sure I can leave this in here

File: doc/source/getting_started/install.rst (hunk 1 above)
Comment (mroeschke) [reply doc/source/getting_started/install.rst]: I realized we have a `pandas[test]` extra that covers all the dependencies so I added it here.

**Review by lithomas1 — APPROVED** [review]

**Pull request merged**